cmd/cri-containerd/options/options.go

```diff
@@ -35,6 +35,17 @@ type ContainerdConfig struct {
 	ContainerdSnapshotter string `toml:"snapshotter"`
 	// ContainerdEndpoint is the containerd endpoint path.
 	ContainerdEndpoint string `toml:"endpoint"`
+	// Runtime is the runtime to use in containerd. We may support
+	// other runtimes in the future.
+	Runtime string `toml:"runtime"`
+	// RuntimeEngine is the name of the runtime engine used by containerd.
+	// We may support other runtime engines in the future.
+	RuntimeEngine string `toml:"runtime_engine"`
+	// RuntimeRoot is the directory used by containerd for runtime state.
+	RuntimeRoot string `toml:"runtime_root"`
+	// RuntimeSystemdCgroup is a flag for enabling systemd cgroup support
+	// in the selected runtime engine.
+	RuntimeSystemdCgroup bool `toml:"runtime_systemd_cgroup"`
```

Comment:
I think we could remove this now.

Reply:
yeah I'll remove it on the rebase

Reply:
well I'll change it to ContainerdSystemdCgroup